crates/wassette/src/lib.rs

```diff
@@ -2187,4 +2251,40 @@ permissions:
 
         Ok(())
     }
+
+    #[test(tokio::test)]
+    async fn test_copy_to_rename_fast_path() -> Result<()> {
```

Comment:
Nit: maybe "happy_path" instead of "fast_path" I stared at this for like 30s before I figured out why it was called this